tests/__snapshots__/files.test.js.snap

```diff
@@ -190,18 +190,18 @@ class test {
        * Replaces text in a string, using a regular expression or search string.
        *
        * @param {string | RegExp} searchValue A string to search for.
-       * @param {string | (substring: string, ...args: any[]) => string} replaceValue A
-       *   string containing the text to replace for every successful match of
-       *   searchValue in this string.
-       * @param {string | (substring: string, ...args: any[]) => string} A_big_string_for_test A
-       *   string containing the text to replace for every successful match of searchValue in
-       *   this string.
-       * @param {string | (substring: string, ...args: any[]) => string} replaceValue A_big_string_for_test
-       *   string containing the text to replace for every successful match of
-       *   searchValue in this string.
-       * @param {string | (substring: string, ...args: any[]) => string} A_big_string_for_test A_big_string_for_test
-       *   string containing the text to replace for every successful match of searchValue in
-       *   this string.
+       * @param {string | (substring: string, ...args: any[]) => string} replaceValue
+       * A string containing the text to replace for every successful match of
+       * searchValue in this string.
+       * @param {string | (substring: string, ...args: any[]) => string} A_big_string_for_test
+       * A string containing the text to replace for every successful match of
+       * searchValue in this string.
+       * @param {string | (substring: string, ...args: any[]) => string} replaceValue
+       * A_big_string_for_test string containing the text to replace for every
+       * successful match of searchValue in this string.
+       * @param {string | (substring: string, ...args: any[]) => string} A_big_string_for_test
+       * A_big_string_for_test string containing the text to replace for every
+       * successful match of searchValue in this string.
        * @returns {StarkStringType & NativeString}
```

Comment:
print width not correct

Reply:
No, the print width is correct. It only seems too short because the `@param` line is too long. I know that the old code simply extended the print width in that case but I considered this bug.